extensions/wikia/FacebookClient/FacebookClient.setup.php

```diff
@@ -34,7 +34,6 @@
 $wgAutoloadClasses['FacebookMapModel'] =  $dir . 'FacebookMapModel.class.php';
 $wgAutoloadClasses['FacebookClientFactory'] =  $dir . 'FacebookClientFactory.php';
 $wgAutoloadClasses['FacebookClientController'] =  $dir . 'FacebookClientController.class.php';
-$wgAutoloadClasses['SpecialFacebookConnectController'] =  $dir . 'SpecialFacebookConnectController.class.php';
```

Comment:
And `fbconnect-preferences-connected-error` is not being used anymore, which is my bad for not removing it :/